src/MoneyFormatterTrait.php

```diff
@@ -25,6 +25,17 @@ trait MoneyFormatterTrait
      */
     public function format($locale = null, Currencies $currencies = null, $style = NumberFormatter::CURRENCY)
     {
+        if (config('money.customFormatter')) {
+            $numberFormatter = new NumberFormatter($locale ?: static::getLocale(), $style);
+
+            $formatter = app(config('money.customFormatter'), [
+                'formatter' => $numberFormatter,
+                'currencies' => $currencies ?: static::getCurrencies(),
+            ]);
```

Comment:
Hi @dalholm 
We don't know what kind of dependencies the formatter will use.
We just have to instantiate:

```php
$formatter = app(config('money.customFormatter'));
```